ouroboros-consensus-cardano-tools/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -200,17 +213,26 @@ mkSomeConsensusProtocolCardano NodeByronProtocolConfiguration {
         Praos.ProtocolParamsBabbage {
           -- This is /not/ the Babbage protocol version. It is the protocol
           -- version that this node will declare that it understands, when it
-          -- is in the Babbage era. Since Babbage is currently the last known
-          -- protocol version then this is also the Babbage protocol version.
-          Praos.babbageProtVer =
-            if npcTestEnableDevelopmentHardForkEras
-            then ProtVer 7 0  -- Advertise we can support Babbage
-            else ProtVer 6 0, -- Otherwise we only advertise we know about (the second) Alonzo
+          -- is in the Babbage era.
+          Praos.babbageProtVer = ProtVer 7 0,
           Praos.babbageMaxTxCapacityOverrides =
             TxLimits.mkOverrides TxLimits.noOverridesMeasure
         }
-        -- ProtocolParamsTransition specifies the parameters needed to transition between two eras
-        -- The comments below also apply for the Shelley -> Allegra and Allegra -> Mary hard forks.
+        Praos.ProtocolParamsConway {
+          -- This is /not/ the Conway protocol version. It is the protocol
+          -- version that this node will declare that it understands, when it
+          -- is in the Conway era.
+          Praos.conwayProtVer =
+            if npcTestEnableDevelopmentHardForkEras
+            then ProtVer 8 0  -- Advertise we can support Conway
+            else ProtVer 7 0, -- Otherwise we only advertise we know about Babbage
```

Comment:
Babbage will have a hardfork with protocol version `8`, so Conway is expected to start with `9`

```suggestion
            then ProtVer 9 0  -- Advertise we can support Conway
            else ProtVer 8 0, -- Otherwise we only advertise we know about Babbage
```